pkg/distributor/distributor_test.go

```diff
@@ -3882,32 +4059,37 @@ func makeFloatHistogramTimeseries(seriesLabels []string, timestamp int64, histog
 	}
 }
 
-func expectedResponse(start, end int, histograms bool) model.Matrix {
+func expectedResponse(start, end int, histograms bool, metrics ...string) model.Matrix {
+	if len(metrics) == 0 {
+		metrics = []string{"foo"}
+	}
```

Comment:
[nit] Have you considered always specifying it? I understand it may require many changes to callers, but would make the logic in the caller easier to follow because it's not based on the assupmption that "no metrics" mean "foo" metric.

Reply:
i don't mind. I think it will make the tests easier to follow. done in 69b2c0ffb4a57d4950d6157f242b8c3bfb092947